packages/interop-tests/src/__test__/interop.test.ts

```diff
@@ -43,13 +40,17 @@ const serverConfig = defaultTestConfig({
   networkConfiguration: {
     chainNetworkID: chainId
       ? parseInt(chainId)
-      : defaultTestConfig().networkConfiguration.chainNetworkID
+      : defaultTestWalletConfig().networkConfiguration.chainNetworkID
   },
   chainServiceConfiguration: {
     attachChainService: true,
     provider: rpcEndpoint,
     pk: TEST_ACCOUNTS[0].privateKey
-  }
+  },
+  // Currently the browser wallet crashes if it receives the same objective again
+  // To avoid this we prevent the wallet from retrying objectives by using
+  // a really large poll value (one hour)
```

Comment:
Is there an issue around this?

Reply:
Haha. I see the issue, prompted by a review comment on the exact same disclaimer text somewhere else...